lib/board.rb

```diff
@@ -0,0 +1,34 @@
+class Board
+  attr_accessor :arr
+
+  def initialize
+    @arr = (1..9).to_a
+  end
+
+  def print
+    puts " #{@arr[0]} | #{@arr[1]} | #{@arr[2]} "
+    puts '---+---+---'
+    puts " #{@arr[3]} | #{@arr[4]} | #{@arr[5]} "
+    puts '---+---+---'
+    puts " #{@arr[6]} | #{@arr[7]} | #{@arr[8]} "
+    puts "\n"
```

Comment:
- [ ] No `puts/print` and `get.chomp` outside `main.rb` file. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone3/only_main_print.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59565018/))